### PR TITLE
Update rubygem-foreman_discovery to 16.1.2 (CP 2.1 RPM)

### PR DESCRIPTION
**Pull request by lzap**
Branch:

* 2.1

### CODE REVIEW

File: packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 Summary:    MaaS Discovery Plugin for Foreman
 Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    %{mainver}
+Version:    16.1.2
```
Comment (evgeni) [inline packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec:20]: As in the `develop` PR, please use the macro, or kill it (for a stable branch, I'd prefer a smaller diff)

**Review by evgeni — CHANGES_REQUESTED** [review]

File: packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec (hunk 1 above)
Comment (lzap) [reply packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec:20]: Removed.

**Review by evgeni — APPROVED** [review]

**Comment by evgeni** [comment]
meh, el8, koji is being stupid again.

**Comment by evgeni** [comment]
[test rpm]

**Pull request merged**